pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -7464,11 +7465,62 @@ spec:
 			},
 			expectedError: `expected workspace "not-source" to be provided by pipelinerun for pipeline task "resolved-pipelinetask"`,
 		},
+		{
+			name: "failure propagating workspaces using scripts",
```

Comment:
could you please use yamls for the tests, it'd be easier to read through it 🙏🏾 

this would involve updating the other 2 test cases that already exist in this function, but it's worth switching over imo

Reply:
Yes, I will do that.